api/app/signals/apps/email_integrations/models.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib.gis.db import models
+
+
+class EmailTemplate(models.Model):
+    SIGNAL_CREATED = 'signal_created'
+    SIGNAL_STATUS_CHANGED_AFGEHANDELD = 'signal_status_changed_afgehandeld'
+    SIGNAL_STATUS_CHANGED_INGEPLAND = 'signal_status_changed_ingepland'
+    SIGNAL_STATUS_CHANGED_HEROPEND = 'signal_status_changed_heropend'
+    SIGNAL_STATUS_CHANGED_OPTIONAL = 'signal_status_changed_optional'
+
+    KEYS_CHOICES = [
+        (SIGNAL_CREATED, 'Send mail signal created'),
+        (SIGNAL_STATUS_CHANGED_AFGEHANDELD, 'Send mail signal handled'),
+        (SIGNAL_STATUS_CHANGED_INGEPLAND, 'Send mail signal scheduled'),
+        (SIGNAL_STATUS_CHANGED_HEROPEND, 'Send mail signal reopened'),
+        (SIGNAL_STATUS_CHANGED_OPTIONAL, 'Send mail optional')
+    ]
+
+    key = models.CharField(max_length=100, choices=KEYS_CHOICES, db_index=True)
+
+    title = models.CharField(max_length=255)
+    body = models.TextField()
```

Comment:
Maybe add a help text explaining how this field can be used? This will also show up in the admin when creating or editing an EmailTemplate.

Reply:
Added a helper.